build.sh

```diff
@@ -18,21 +18,22 @@ ARGS=$*
 # script, and that this script resides in the repo dir!
 REPODIR=$(cd $(dirname $0); pwd)
 
-VALIDARGS="clean libnvstrings nvstrings libcudf cudf dask_cudf benchmarks -v -g -n --allgpuarch -h"
-HELP="$0 [clean] [libcudf] [cudf] [dask_cudf] [benchmarks] [-v] [-g] [-n] [-h]
-   clean        - remove all existing build artifacts and configuration (start
-                  over)
-   libnvstrings - build the nvstrings C++ code only
-   nvstrings    - build the nvstrings Python package
-   libcudf      - build the cudf C++ code only
-   cudf         - build the cudf Python package
-   dask_cudf    - build the dask_cudf Python package
-   benchmarks   - build benchmarks
-   -v           - verbose build mode
-   -g           - build for debug
-   -n           - no install step
-   --allgpuarch - build for all supported GPU architectures
-   -h           - print this text
+VALIDARGS="clean libnvstrings nvstrings libcudf cudf dask_cudf benchmarks -v -g -n -x --allgpuarch -h"
```

Comment:
Add `--disable_nvtx` to the `VALIDARGS` variable.

Reply:
Done